### PR TITLE
Running actions list with poll api(temporary)

### LINKED ISSUE #23: Implement Stdout in API
https://github.com/launchrctl/web/blob/main/server/api.go#L42

Add terminal output. In simplest possible option for now.

### PR DESCRIPTION
**Pull request by iberdinsky-skilld**
Closes #23

### CODE REVIEW

File: server/api.go [hunk 1]
```diff
@@ -39,8 +40,23 @@ func (l *launchrServer) GetRunningActionStreams(w http.ResponseWriter, _ *http.R
...
-	panic("not implemented")
-	//_ = ri.Action.GetRunEnvironment().Logs(r.Context(), "", "")
+	outputFile, err := os.ReadFile(fmt.Sprintf("%s-out.txt", id))
+	if err != nil {
+		if os.IsNotExist(err) {
+			sendError(w, http.StatusNotFound, fmt.Sprintf("Output file associated with actionId %q not found", id))
+		} else {
+			sendError(w, http.StatusInternalServerError, "Error accessing file")
+		}
+		return
```
Comment (iignatevich) [inline server/api.go:50]: send custom error or panic maybe ?

Comment (iberdinsky-skilld) [reply server/api.go]: fixed

File: server/streams.go [hunk 2]
```diff
@@ -49,10 +50,16 @@ func (w *wrappedWriter) Write(p []byte) (int, error) {
 	return w.w.Write(p)
 }
 
-func fileStreams() *webCli {
+func fileStreams(actionId ActionId) *webCli {
 	// @todo wrap writer to prepend info
-	outfile, _ := os.Create("out.txt")
-	errfile, _ := os.Create("err.txt")
+	outfile, errO := os.Create(fmt.Sprintf("%s-out.txt", actionId))
```
[the remark this answers is not in the document]
Comment (iberdinsky-skilld) [reply server/streams.go]: fixed

**Comment by koskinpark** [comment]
Have tested locally, it works. But later we have to colorize stdoud somehow

**Pull request merged**